filters/ThirdParty/filter_222_ABPPersian/template.txt

```diff
@@ -409,6 +419,7 @@ _vps-banner*.gif
 ||cloob.com^$third-party
 ||chare.in^$third-party
 ||dabi.ir$~third-party
+||digikala.com^$third-party
```

Comment:
Do I understand correctly that this rule will be added to filter?
If so, then if I'm not wrong it will breaks a website from this issue - https://github.com/AdguardTeam/AdguardFilters/issues/53365

Reply:
forgot to remove. Removed.